test/cfa/ntp_conf_test.rb

```diff
@@ -418,6 +418,17 @@ def ntp_conf(file)
       expect(file.content).to include("restrict -4 default notrap\n")
     end
   end
+
+  context "when creating a new record from scratch" do
+    it "sets values correctly" do
+      record = CFA::NtpConf::RestrictRecord.new
+      record.value = "192.168.0.1"
+      record.options = ["default"]
```

Comment:
I complain :) default is not an option but an address, it is used when the ip protocol version is used just because the augeas lens in use does not covers -4 option, but in this case as you are using an IP address then I prefer to use real options like `nomodify notrap nopeer noquery`